pkg/azureconstants/azure_constants.go

```diff
@@ -106,6 +106,9 @@ const (
 	CRIUpdateRetryDuration = time.Duration(1) * time.Second
 	CRIUpdateRetryFactor   = 3.0
 	CRIUpdateRetryStep     = 5
+
+	DefaultMountRecoveryBackoff      = time.Duration(2) * time.Minute
+	DefaultMountRecoveryBackoffScale = 2
```

Comment:
These appear to be unused now. Do we still need them?

Reply:
These were remnants from the previous back offs. I will remove them 👍